Contributor.md

```diff
@@ -4,6 +4,12 @@
 
 ---
 
+- Name: Jeel Pujara
```

Comment:
Hey Jeel please respect other's post and move your post at the end of the markdown